ci/pipeline.yml

```diff
@@ -37,6 +37,24 @@ jobs:
       vars_files:
       - autoscaler-manifests/bosh/varsfiles/development.yml
       - terraform-secrets/terraform.yml
+  on_success:
+    put: slack
+    params:
+      text: |
+        :white_check_mark: Successfully deployed CF App Autoscaler on development
+        <$ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
+      channel: '#cg-platform-news'
```

Comment:
We have a credhub variable that I'd recommend here: `((slack-channel-success))`

Reply:
Added with subsequent commit, thanks!